### PR TITLE
Add auto deploy of docs to github pages

### PR DESCRIPTION
**Pull request by TheZoq2**
I have no idea if I did this correctly, this is my first time using GH actions and I just copy pasted stuff, but in theory, it should build the documentation for the engine and deploy that to gh-pages. To make things convenient, I renamed docs.html to index.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,10 +21,18 @@ command.
...
+RIBS comes with a sample game which you can modify to make it your own. To run it,
+simply call
 
 ```bash
 python3 game.py
 ```
 
-Now you are ready to create your game!
\ No newline at end of file
+To modify the game, open game.py and make your own changes
```
Comment (sornas) [inline README.md]:
```suggestion
To modify the game, open game.py and make your own changes.
```

File: .github/workflows [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Gen dummy page
+        run: |
+          cd docs
+          pip install pygments
+          ./doc-generator.py
```
Comment (sornas) [inline .github/workflows]: `doc-generator.py` isn't currently set as executable and is missing a shebang.

**Comment by TheZoq2** [comment]
Good catch, fixed

File: .github/workflows [hunk 3]
```diff
@@ -17,7 +17,7 @@ jobs:
         run: |
           cd docs
           pip install pygments
-          ./doc-generator.py
+          python doc-generator.py
```
Comment (sornas) [inline .github/workflows]:
```suggestion
          python3 doc-generator.py
```
`python` might be linked to Python 2, unfortunately.

**Pull request merged**